i18n-embed/Cargo.toml

```diff
@@ -24,6 +24,7 @@ fluent-syntax = { version = "0.11", optional = true }
 gettext_system = { package = "gettext", version = "0.4", optional = true }
 i18n-embed-impl = { version = "0.8", path = "./i18n-embed-impl", optional = true }
 intl-memoizer = "0.5"
+itertools = "0.10"
```

Comment:
Is this still being used?

Reply:
I used that in an earlier version but could then use the builtin `.chain()` method. It's not being used.